### ISSUE TITLE
Pycharm2021.3 Community Edition is not compatible with the plugin

### ISSUE THREAD

**Issue by TARI0510** [tonybaloney/pycharm-security#308]
Cannot load this plugin using pycharm2021.3 commnunity edition

```
19:37	Plugin error: Plugin 'Python Security' (version 1.25.0) was explicitly marked as incompatible with the current version of the IDE
```
![image](https://user-images.githubusercontent.com/39155974/144839882-f7e33b18-7d24-4c4e-893e-dcd898817996.png)
![image](https://user-images.githubusercontent.com/39155974/144840244-a504fd79-759b-4d51-89c8-5df46b2ad0b3.png)

 - Plugin Version: 1.25.0
 - PyCharm Version: Build #PC-213.5744.248, built on November 30, 2021 Runtime version: 11.0.13+7-b1751.19 amd64
 - Python Version: 3.7

**Comment by tonybaloney** [comment]
Thanks for the report, the plugin is marked as requiring "213" (2021.3) and above. I don't understand why its giving this error.

There are some breaking changes in the PyCharm APIs from 2021.3 so I have to deprecate 2021.2 and older. 

Will investigate

**Comment by TARI0510** [comment]
Yes, for example, many rules need `typeContext` after upgrading to 1.25.0, so some of my own rules don't work, I have fixed it for myself.
I used the default `build.gradle` configuration of version 1.24.2, and it worked fine, but upgrading to version 1.25.0 did not, although I try to change `plugin.xml` `<idea-version>`'s `since-build` and `until-build`
Looking forward to fixing, thank~

**Comment by tonybaloney** [comment]
<img width="444" alt="screenshot 2021-12-07 at 12 29 33" src="https://user-images.githubusercontent.com/1532417/144949657-605dec22-e28c-46d4-b074-09a45d53fd3b.png">
I've tried fiddling with those settings too. This makes no sense!

**Comment by tonybaloney** [comment]
Fixed in 1.25.1
